### ISSUE TITLE
possible to enable jit on jailbroken device with utm windows 

### ISSUE THREAD

**Issue by leo100410** [fritzlb/iOS17-JIT-WIN#77]
could you install windows in utm on jailbroken and plug an unjailbroken device into it and enable jit on the unjailbroken device on ios 17 with the emulated windows?

**Comment by fritzlb** [comment]
I do not know. Maybe. 
I don't have a jailbroken machine (and if I had one I wouldn't need this script)
Cool Idea though, let me know if you get it working :)

**Comment by leo100410** [comment]
Yeah I’ll try it but it’s so complicated with windows because only win7 is working so would the script run on Linux because it’s easier to install in utmAm 05.06.2024 um 18:50 schrieb fritzlb ***@***.***>:﻿
I do not know. Maybe.
I don't have a jailbroken machine (and if I had one I wouldn't need this script)
Cool Idea though, let me know if you get it working :)

**Comment by leo100410** [comment]
Also you said from the pojav dev release 1.20 is working but I can’t click on the files in the link how to download them
